### PR TITLE
Activity date fields -> datepicker

### PR DESCRIPTION
**Pull request by colemanw**
Overview
----------------------------------------
Switches the activity date and case start date fields to use the new datepicker widget.
See https://lab.civicrm.org/dev/core/issues/561

### CODE REVIEW

**Comment by colemanw** [comment]
Ping @seamuslee001

File: CRM/Case/Form/Activity.php [hunk 1]
```diff
@@ -272,12 +272,13 @@ public function buildQuickForm() {
...
-      );
+      if ($this->_action == CRM_Core_Action::UPDATE) {
+        $this->getElement('activity_date_time')->freeze();
+      }
+      // Fixme: what's the justification for this? It seems like it is just re-adding an option in case it is the default and disabled.
+      // Is that really a big problem?
+      $this->_encounterMedium = CRM_Core_DAO::getFieldValue('CRM_Activity_DAO_Activity', $this->_activityId, 'medium_id');
       if (!array_key_exists($this->_encounterMedium, $encounterMediums)) {
         $encounterMediums[$this->_encounterMedium] = CRM_Core_OptionGroup::getLabel('encounter_medium',
           $this->_encounterMedium,
```
Comment (jitendrapurohit) [inline CRM/Case/Form/Activity.php]: Should we also remove the param `activity_date_time_time` from https://github.com/civicrm/civicrm-core/blob/master/CRM/Case/Form/Activity.php#L408? Currently, on saving a case activity, I get a notice error as follow -

image

Other changes seems to work fine on my testing 👍

Comment (mattwire) [reply CRM/Case/Form/Activity.php]: The whole line 408 and the comment above can be removed. They are not required anymore.

**Comment by colemanw** [comment]
Thanks @jitendrapurohit - I've taken @mattwire's suggestion and marking this merge-on-pass based on your review.

**Comment by seamuslee001** [comment]
Jenkins re test this please

**Comment by seamuslee001** [comment]
Merging as per the tag

**Pull request merged**